SpecEasy/GenericSpec.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
 using System.Reflection;
-using Rhino.Mocks;
-using Rhino.Mocks.Interfaces;
+using NSubstitute;
 using TinyIoC;
 
 namespace SpecEasy
 {
-    public class Spec<TUnit> : Spec
+    public class Spec<TUnit> : Spec where TUnit: class

```

Comment:
Unfortunate, but there's always the option of testing value types  using a vanilla `Spec`.